src/Concordium/Client/Runner.hs

```diff
@@ -125,13 +124,17 @@ import           System.Directory
 import           System.FilePath
 import qualified System.Console.ANSI                 as ANSI
 import           Text.Printf
-import           Text.Read (readMaybe)
+import           Text.Read (readMaybe, readEither)
 import Data.Time.Clock (addUTCTime, getCurrentTime, UTCTime)
 import Data.Ratio
 import Codec.CBOR.Write
 import Codec.CBOR.Encoding
 import Codec.CBOR.JSON
 import Control.Arrow (Arrow(second))
+import Concordium.GRPC2
+import Concordium.Types (AccountIdentifier(AccAddress))
+import Concordium.Client.GRPC2
```

Comment:
Concordium.Client.GRPC2 will replace Concordium.Client.GRPC post migration and it is a heavily referenced module in this file. Qualifying it will result in a lot of prefixes, so I think it may make the code more terse. `Concordium.Types` is already qualified, so that import should definitely be removed. I'll qualify the latter :-) thanks.